docs/docs/getting-started/5-streaming.md

```diff
@@ -0,0 +1,76 @@
+# A Quick Example: Live Stream Alert 
+
+A quick example to get used to setting up detectors and asking good questions: set up a monitor on a live stream. 
+
+## Requirements
+
+- [Groundlight SDK](/docs/installation/) with Python 3.7 or higher
+- The video ID of a YouTube live stream you'd like to monitor
+
+## Installation
+
+Ensure you have Python 3.7 or higher installed, and then install the Groundlight SDK and OpenCV library:
+
+```bash
+pip install groundlight pillow ffmpeg yt-dlp typer
+```
+
+## Creating the Application
+
+1. Save this command as a shell script `get_last_frame.sh`:
+
+```
+#!/bin/bash
+
+ffmpeg -i "$(yt-dlp -g $1 | head -n 1)" -vframes 1 last.jpg -y
+``` 
+
+This will download the last frame from a YouTube live stream and save it to a local file `last.jpg`. 
+
+2. Log in to the [Groundlight application](https://app.groundlight.ai) and get an [API Token](api-tokens).
+
+3. Next, we'll write the Python script for the application.
+
+```python notest
+import os
+import subprocess
+import typer
+from groundlight import Groundlight
+from PIL import Image
+
+GROUNDLIGHT_API_TOKEN=os.getenv("GROUNDLIGHT_API_TOKEN")
+
+
+def main(*, video_id: str = None, detector_name: str = None, query: str = None, confidence: float = 0.75, wait: int = 60):
+    """
+    Run the script to get the stream's last frame as a subprocess, and submit result as an image query to a Groundlight detector
+    :param video_id: Video ID of the YouTube live stream
+    :param detector_name: Name for your Groundlight detector
+    :param query: Question you want to ask of the stream (we will alert on the answer of NO)
+    """
+    gl = Groundlight(api_token=GROUNDLIGHT_API_TOKEN)
```

Comment:
It should automatically be set inside of `Groundlight` already, but if this is just to be illustrative, that is ok